### PR TITLE
Migrate to 0.8.0

### PR DESCRIPTION
**Pull request by jsperezg**
#### :tophat: What? Why?
This PR adapts initiatives to 0.8.0.

### CODE REVIEW

**Comment by mrcasals** [comment]
There are conflicts, can you fix them please? 😄

**Comment by josepjaume** [comment]
It's strange, I'm seeing way to many differences with master. Looks like the PR is not correctly rebased. Right?

**Comment by jsperezg** [comment]
@josepjaume 

I started this branch prior to all changes and fixes made before it was deployed in metadecid. In addition I also added your byebug settings and I started fixing offenses in all files that need an adjustment.

**Comment by josepjaume** [comment]
@jsperezg nice! There's some failing specs though. It'd be great to have this merged, as I'll be able to update *decidim-barcelona* to `0.8.3` then (we have some [strange failing test cases](https://github.com/AjuntamentdeBarcelona/decidim-barcelona/pull/178) related to `decidim-initiatives` in there right now).

Good work BTW!

File: .bash_history [hunk 1]
```diff
@@ -0,0 +1,17 @@
+rm -Rf spec/decidim_dummy_app/
```
Comment (mrcasals) [inline .bash_history]: Remove this file

**Pull request merged**